src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariable.java

```diff
@@ -0,0 +1,204 @@
+/*
+UnusedLocalVariable
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+import java.util.ArrayList;
+import java.util.function.Predicate;
+
+public class InputUnusedLocalVariable {
```

Comment:
Please add test cases for try with resources (passing and violation). Make sure to also have cases with multiple resources.

Reply:
Disregard this if we are not supporting it.

Reply:
Resources are automatically closed (unless you call `System.exit(..)` or turn off the power) so they cannot go unused, I didn't add them to the list of accepted tokens as there is no need for checking them, added the test case for them.

Reply:
Resolved.